tensorflow/python/ops/ragged/ragged_tensor.py

```diff
@@ -1637,7 +1637,8 @@ def from_sparse(cls, st_input, name=None, row_splits_dtype=dtypes.int64):
 
     Example:
 
-    >>> st = tf.sparse.SparseTensor(indices=[[0, 0], [0, 1], [0, 2], [1, 0], [3, 0]],
+    >>> st = tf.sparse.SparseTensor(indices=
```

Comment:
Better extract the list passed to the `indices` to a value outside and then use that here

Reply:
Committed as suggested, thanks.